content/blog/2024-12-21-how-do-i-...-in-dt-5.0/index.md

```diff
@@ -0,0 +1,508 @@
+  for perceptual brightness adjustments (the "global brilliance" slider is
+  on the Quick Access Panel by default).
+
+To adjust saturation, you can use
+
+- the _global vibrance_, _chroma grading_, or _saturation grading_ sliders in
+  [_color balance rgb_](https://darktable.org/usermanual/en/module-reference/processing-modules/color-balance-rgb/)
+  (on the Quick Access Panel by default).
+
+- the [_velvia_](https://darktable.org/usermanual/en/module-reference/processing-modules/velvia/) module
+
+</details>
+
+<details><summary><b>adjust shadows/highlights</b></summary>
+
+To open shadows or improve details in highlights, use the
+[_tone equalizer_](https://darktable.org/usermanual/en/module-reference/processing-modules/tone-equalizer/).
+
+On its _masking_ tab, click the two magic wands, and fine-tune the
+mask brightness with the _mask exposure compensation_ and _mask
+contrast compensation_ sliders.  Then move your mouse over areas of
+the image you want to adjust and scroll to brighten or darken areas of
+the image with that brightness.
+
+</details>
+
+<details><summary><b>make more extreme adjustments</b></summary>
+
+Most sliders in darktable have both soft and hard limits.  The soft
+limit is the range displayed by default (e.g. -3 to +4 EV adjustment
+in the _exposure_ module).  The hard limit is the maximum which
+darktable actually supports, but which is usually too extreme and
+results in insufficiently precise control using the slider (e.g. -18
+to +18 EV adjustment for _exposure_, which is an amount only useful
+for merged HDR sequences).  You can override the soft limits by
+
+- holding Ctrl+Shift while dragging the slider or using arrow keys
+- scrolling while in the left-most or right-most 10% of the slider
+- right-clicking on the slider and typing in the desired value
+
+In cases where there is no separate hard limit, or the hard limit is
+_still_ too small, you can usually strengthen whichever effect is not
+enough by adding a second (third, etc.)
+[instance](https://darktable.org/usermanual/en/darkroom/processing-modules/multiple-instances/)
+of the module.
+
+</details>
+
+<details><summary><b>add clarity</b></summary>
+
+What other programs call clarity is an adjustment of local contrast.
+You can do this in darktable with the
+[_local contrast_](https://darktable.org/usermanual/en/module-reference/processing-modules/local-contrast/)
+module (on the [_Quick Access Panel_](https://darktable.org/usermanual/en/darkroom/organization/quick-access-panel/)
+by default) or by starting with the "clarity" preset in the
+[_contrast equalizer_](https://darktable.org/usermanual/en/module-reference/processing-modules/contrast-equalizer/)
+module (quite strong) or one of the "local contrast" presets in the
+[_diffuse and sharpen_](https://darktable.org/usermanual/en/module-reference/processing-modules/diffuse/)
+module (generally very subtle).
+
+</details>
+
+<details><summary><b>apply Camera Calibration settings</b></summary>
```

Comment:
This header doesn't feel accurate. I don't get it.

Reply:
Trying to use Lr terms, but since I don't use Lr....  Mapping the Camera Calibration panel to a dt equivalent.